src/components/StoragePartitionedBanner.tsx

```diff
@@ -0,0 +1,210 @@
+import { useState, useEffect, useRef, useMemo } from "react";
+import styled from "styled-components";
+import { motion, AnimatePresence } from "framer-motion";
+import { InfoCircle, X as XClose } from "@untitled-ui/icons-react";
+import { useEmbedded } from "~utils/embedded/embedded.hooks";
+import {
+  PARTITIONED_STORAGE_BANNER_DISMISSAL_KEY,
+  PARTITIONED_STORAGE_BANNER_EVENT,
+  PARTITIONED_STORAGE_BANNER_MESSAGE
+} from "~iframe/storage/unpartitioned-storage/unpartitioned-storage.utils";
+
+// Styled components for the banner
+const BannerWrapper = styled(motion.div)`
+  position: fixed;
```

Comment:
I think it would be better if we make it non-dismissible, appears from the bottom but without `position: fixed` so that it doesn't cover the rest of the UI, and includes a button to re-request access.

The reason for this is that people might reject the permission without being aware of the consequences, and if they then dismiss the banner, they won't be able to get it back or to easily fix it (other than trying to reload the page).

Also, once the UI PR is merged, we probably want to have this appear in all screens, so maybe a better place to put it is `src/components/page/page.component.tsx`.